src/solvers/smt2/smt2_conv.cpp

```diff
@@ -2137,20 +2137,25 @@ void smt2_convt::convert_expr(const exprt &expr)
     else if(quantifier_expr.id() == ID_exists)
       out << "(exists ";
 
-    out << "( ";
+    out << '(';
+    bool first = true;
     for(const auto &bound : quantifier_expr.variables())
     {
-      out << "(";
+      if(first)
+        first = false;
+      else
+        out << ' ';
```

Comment:
Is this space actually needed or is it also redundant?

Reply:
Yes, you'd get ")(" otherwise. Syntactically valid, but not pretty.

Reply:
That makes sense. I wasn't aware if the goal was making this prettier or just shorter.